vm/actor/src/builtin/market/mod.rs

```diff
@@ -563,10 +569,9 @@ impl Actor {
                 }
                 let deal = deal.unwrap();
 
-                assert_eq!(
-                    deal.provider, miner_addr,
-                    "caller is not the provider of the deal"
-                );
+                if deal.provider != miner_addr {
+                    return Err(actor_error!(ErrIllegalState; "caller {} is not the provider {} of deal {}", miner_addr, deal.provider, id));
```

Comment:
same here as the point above, formatting will be off (especially with how long this line is) and affects the rest of the file and will be confusing for people modifying in the future